js/ui.js

```diff
@@ -877,11 +882,12 @@ SIM.UI = {
 
     simulateRow: function(tr, updateFn) {
         var view = this;
-        var ehp = tr.find('td:nth-last-of-type(5)');
-        var psurv = tr.find('td:nth-last-of-type(4)');
-        var pullvar = tr.find('td:nth-last-of-type(3)');
-        var dps = tr.find('td:nth-last-of-type(2)');
-        var tps = tr.find('td:last-of-type');
+        var ehp = tr.find('td:nth-last-of-type(6)');
```

Comment:
This tr.find(X) lookup method is kind of gross, but CBA change it now (not yours, just the way it's grown over time)